worte/usr/share/asteroid-launcher/watchfaces/worte.qml

```diff
@@ -0,0 +1,110 @@
+            var hoursList = ["<b>zwölf</b>", "<b>ein</b>", "<b>zwei</b>", "<b>drei</b>", "<b>vier</b>", "<b>fünf</b>", "<b>sechs</b>", "<b>sieben</b>", "<b>acht</b>", "<b>neun</b>", "<b>zehn</b>", "<b>elf</b>"]
+            var minutesFirst = [false, true, true, true, false, true, false, true, false, true, true, true, false]
+            var nextHour = [false, false, false, false, false, false, false, false, false, true, true, true, true]
+            var hourSuffix = [true, true, false, false ,true, false, true, false, true, false, false, false, true]
+
+            var minutes = Math.round(time.getMinutes()/5)
+            var hours = (time.getHours() + (nextHour[minutes] ? 1 : 0)) % 12
+
+            var start = "<p style=\"text-align:center\">"
+            var newline = "<br>"
+            var end = "</p>"
+            
+            if (hourSuffix[minutes]) {
+                if (minutesFirst[minutes]) {
+                    var generatedString = minutesList[minutes].toUpperCase() + newline + hoursList[hours].toUpperCase() +" UHR"
+                    } else {
+                    var generatedString = hoursList[hours].toUpperCase()+" UHR" + newline + minutesList[minutes].toUpperCase()}
+                } else {
+                    if (minutesFirst[minutes]) {
+                        var generatedString = minutesList[minutes].toUpperCase() + newline + hoursList[hours].toUpperCase()
+                        } else {
+                        var generatedString = hoursList[hours].toUpperCase() + newline + minutesList[minutes].toUpperCase()
+                    }
+                }
+            return start + generatedString + end
+        }
+
+        id: timeDisplay
+
+        textFormat: Text.RichText
+
+        font.pixelSize: parent.height*0.15
+        font.weight: Font.Light
+        lineHeight: 0.85
+        color: "white"
+        style: Text.Outline; styleColor: "#80000000"
+        horizontalAlignment: Text.AlignHCenter
+        anchors {
+            verticalCenter: parent.verticalCenter
+            left: parent.left
+            right: parent.right
+        }
+        text: generateTime(wallClock.time)
+    }
+
+    Text {
+        id: dateDisplay
+
+        font.pixelSize: parent.height*0.07
+        color: "white"
+        style: Text.Outline; styleColor: "#80000000"
+        opacity: 0.8
+        horizontalAlignment: Text.AlignHCenter
+        anchors {
+            topMargin: 10
+            top: timeDisplay.bottom
+            left: parent.left
+            right: parent.right
+        }
+        text: wallClock.time.toLocaleString(Qt.locale(), "<b>dddd</b> d MMM")
+    }
+
+    Connections {
+        target: localeManager
+        onChangesObserverChanged: {
+            timeDisplay.text = Qt.binding(function() { return generateTime(wallClock.time) })
+            dateDisplay.text = Qt.binding(function() { return wallClock.time.toLocaleString(Qt.locale(), "<b>dddd</b> d MMM") })
+        }
+    }
+}
```

Comment:
One last thing. Your commit log is really hard to read because you made your PR with every changes you did.
Before making a pull request, please clean your git log. There are various ways to do that, the most flexible is using an interactive git rebase. I would advise to just recreate the entire commit with one commit per watchface.
If you force push this new commit log to your repository, the pull request will automatically be updated.